Tools/scripts/install-travis-env.sh

```diff
@@ -46,60 +34,18 @@ for pkg in $PYTHON_PKGS; do
     sudo pip -q install $pkg || echo "FAILED INSTALL OF $pkg"
 done
 
-# install some extra packages (for later AVR compiler)
-rsync -av firmware.ardupilot.org::Tools/Travis/*.deb ExtraPackages
-sudo dpkg -i ExtraPackages/*.deb || echo "FAILED INSTALL OF EXTRA DEBS"
-
```

Comment:
I think we still need this code to download the correct packages and install them.  See comment above. 

I agree we can/should update the gcc-avr deb package on there to 4.9